rules/game_process.tex

```diff
@@ -165,11 +165,10 @@ \subsection{Indirect Kick}
 \begin{itemize}
   \item A direct kick is allowed for the duration of the penalty kick, but this rule still applies after this kick.
   \item If a team is awarded a free kick but fails to execute it within the allotted time, the opposing team may attempt to score directly if they play the ball before the team who was originally awarded the free kick does.
-    ("Execute" is intended as "kick and move the ball clearly", as per section \cref{sec:free_kick_execution})
+    (``Execute'' is intended as "kick and move the ball clearly", as per section \cref{sec:free_kick_execution})
```

Comment:
```suggestion
    (``Execute'' is intended as ``kick and move the ball clearly'', as per section \cref{sec:free_kick_execution})
```

Reply:
Fixed that and another hidden double quote, ty